client-service/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.0/apache-maven-3.5.0-bin.zip
```

Comment:
Also there is `client-service/.mvn/wrapper/maven-wrapper.jar` that is binary file, and which also should be removed.